### ISSUE TITLE
Try making any charts that are compared to a national average, use the color of the chart as intrinsic indicators of "goodness"

### ISSUE THREAD

**Issue by LisaGee** [18F/college-choice#531]
So in Average Cost per Year - if the chart reaches above the line, it would be red and at the line, yellow and below the line green.
But for Graduation rate - if the chart for a particular school is above the nat'l average line, green, below, red, etc...

**Comment by LisaGee** [comment]
We'll do a rough version based on real data and see what it looks like before evaluating whether it is worth making this change before launch.

**Comment by saintsoup52** [comment]
Huge +1 from me on this.

**Comment by claremccann** [comment]
Given the data limitations inherent in a lot of these elements, I think red will probably seen by many as potentially misleading or unfair.

**Comment by jjoteal** [comment]
Hi folks,

Here's a very rough mock-up of this, using the green = above average, yellow = about average, red = below average color scheme:

![image](https://cloud.githubusercontent.com/assets/12192706/9450244/91340304-4a75-11e5-9807-2ff238658021.png)


I personally like it a lot and feel as though it sends an instant message to users who are scanning the page whether or not a school is worth looking at or not...

I vote YES on this!

If we go for this, one thing we'll need to determine then are the ranges for "average" -- i.e. should we take the national average +/- 10% on each end and consider that yellow = average? I think just classifying "average" as being right on the line would be too limiting.

**Comment by saintsoup52** [comment]
I really like this!!

The visual reinforces the story the data is telling. I think the buckets should be 1/3, 1/3, 1/3 (same methodology as High/Medium/Low)... therefore ED will not be creating anything new just modifying the visual design...

**Comment by saintsoup52** [comment]
Just following up on this issue... Any chance we can make this happen before launch?

**Comment by barkimedes** [comment]
@jjoteal Will this still be a pretty good experience for red/green color blind folks? I guess it isn't tragic sine they can still glean the information from the charts, but it would still be kinda sad to degrade their experience (assuming the colors we're using now are good for them?)

**Comment by saintsoup52** [comment]
I would also recommend adding the words "Above Average", "About Average" and "Below Average" underneath each metric as you already do on the school profile page.

![image](https://cloud.githubusercontent.com/assets/4397508/11042528/a554085a-86e5-11e5-99e0-ab1fff1fc6fa.png)

**Comment by jjoteal** [comment]
@barkimedes I agree with @saintsoup52. If we go with the red, green, yellow bar graphs, I think it will be very important to ensure that the "Above Average", "About Average" and "Below Average" signifiers accompany each graph. That way those with color blindness will still be able to comprehend the charts. Please note that we'll likely  need to change the color of those labels though so they are NOT green (as that would be confusing with the green used in the bar charts). Once this task is prioritized, happy to take a design crack at this.
